### DISCUSSION TITLE
Enabling Web Crawling rules on Docusaurus(.69) website

### DISCUSSION THREAD

**Discussion by divyabhushan-okta** [facebook/docusaurus#4033] [Show and tell]
Hello all,

I would like to share how I added **robots.txt** file to my Docusaurus2 v.69 site.

> Have you faced the **robots.txt file not found error** after adding the file to your root directory?

I discovered that the *robots.txt* file must not be placed in the *root* directory as suggested in the official document, but in the `static` folder. 
Doing so will copy the file from the `static` folder to the `build` directory and added to your Docusaurus website.
I have a written [blog](https://divya-bhushan.medium.com/where-to-place-the-robots-txt-file-in-your-docusaurus-website-45d18e40e285) on the same and would welcome your thoughts on the same.

Thanks,
Regards,
Divya

**Comment by slorber** [comment]
thanks

yes, anything in /static will be deployed as part of your site, so you can put whatever you want here

**Reply by divyabhushan-okta** [reply]
Got it, thank you.
